kinetic/distribution.yaml

```diff
@@ -5095,17 +5095,17 @@ repositories:
     doc:
       type: git
       url: https://github.com/KITrobotics/iirob_filters.git
-      version: kinetic-devel
+      version: melodic
```

Comment:
The documentation in the readme still says that Kinetic is developed on the branch `kinetic_devel` are you sure you want to switch the branch listing here?

Reply:
Yes. I am moving everything to melodic, but the code is backward compatible. I didn't want to mange 2 branches in the future. I will also change readme.